packages/components/dialog/src/dialog.ts

```diff
@@ -83,25 +141,57 @@ export class Dialog extends ScopedElementsMixin(LitElement) {
     }
   }
 
+  #handleAnimationEnd = (event: AnimationEvent, target?: HTMLElement): void => {
```

Comment:
Why `handle` instead of `on`? The convention I use everywhere is to name event handler with the prefix `#on`; so `#onAnimationEnd`